README.md

```diff
@@ -1,2 +1,34 @@
-# openmm-ml
-High level API for using machine learning models in OpenMM simulations
+# OpenMM-ML
+
+This is a high level API for using machine learning models in OpenMM simulations.  With just a few lines of code, you
+can set up a simulation that uses a standard, pretrained model to represent some or all of the interactions in a system.
+
+The current version is an early release.  The only supported potential functions are ANI1ccx and ANI2x.  They are
```

Comment:
The usual style is with a hyphen: ANI-1ccx an ANI-2x (https://aiqm.github.io/torchani/api.html#module-torchani.models)

Reply:
Added them.